.travis.yml

```diff
@@ -7,7 +7,7 @@ notifications:
 # Run latest 3.x and 2.x next, to get quick compatibility results.
 # Then run the remainder.
 python:
-  - "pypy"
+  - "pypy-5.3.1"
```

Comment:
@Fahad-Alsaidi That looks good (I've edited it slightly). Please could you add it as a comment in .travis.yml, above `- "pypy-5.3.1"`?